File: 1002.cs
using System;

class URI {
    
    static void Main(string[] args) {
 double pi = 3.14159;

        double raio = double.Parse(Console.ReadLine());

        double area = pi * Math.Pow(raio, 2);

        Console.WriteLine("A=" + area.ToString("F4"));
    }
}

File: RUST/1002.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let raio: f64 = input.trim().parse().expect("Invalid input");

    let pi = 3.14159;
    let area = pi * raio * raio;

    println!("A={:.4}", area);
}